src/api/java/Status.java

```diff
@@ -38,7 +38,7 @@
         this.intValue = v;
     }
 
-    public static final Status fromInt(int v)
+    public static Status fromInt(int v)
```

Comment:
None whatsoever, "final" does not make sense when added to static methods (as child classes can override the method anyway).
Annotations should only appear when they have meaningful semantics, and this one does not.